pkg/Microsoft.Windows.Compatibility/Microsoft.Windows.Compatibility.pkgproj

```diff
@@ -49,10 +50,14 @@
     <PrereleaseLibraryPackage Include="System.Text.Encoding.CodePages" />
     <PrereleaseLibraryPackage Include="System.Threading.AccessControl" />
 
+    <!-- External pre-release packages -->
+    <LibraryPackage Include="System.ServiceModel.Primitives">
+      <Version>4.4.1-servicing-25917-01</Version>
```

Comment:
Isn't this stable now? If so we should use the stable version. @StephenBonikowsky did you guys produce a stable version of this yet?

Reply:
We don't have a stable version published to nuget.org yet. Next month...

Reply:
So then we can produce a package with this changes and next month when there is a stable version we can update again right?

Reply:
I'm not sure we will update the preview just for that but we can if needed. We just need to ensure we have a stable version before we can release the compat package stable.

Reply:
Ok, sounds reasonable.